File: URI1013.cpp
#include <iostream>
#include <cstdio>

using namespace std;

int main() {
    int A, B, C, MAIORAB, MAIOR;

    cin >> A, cin >> B, cin >> C;

    MAIORAB = (A + B + abs(A-B))/2;
    MAIOR = (MAIORAB + C + abs(MAIORAB - C))/2;

    printf("%d eh o maior\n", MAIOR);

    return 0;
}

File: 1. Iniciante/URI1013.py
a,b,c = input().split(" ")

A = int(a)
B = int(b)
C = int(c)

MAIORAB = (A + B + abs(A-B))/2
MAIOR = (MAIORAB + C + abs(MAIORAB - C))/2

print("%d eh o maior" %MAIOR)
